feat: fcc-todo-app (#51367)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-localstorage-by-building-a-todo/64e4f01d6c72086e016a8626.md

---
id: 64e4f01d6c72086e016a8626
title: Step 12
challengeType: 0
dashedName: step-12
---

# --description--

> Now you will get the values of the input fields and save them in a `taskObj` object. In addition, there will also be an `id` for each task.

Reviewer: ```suggestion
Next, retrieve the values from the input fields and store them in a `taskObj` object. Each task should also have a unique `id`.
```

> Create a `taskObj` object with `id` as the first property. For the value of that `id` property, get the value of the `titleInput` and convert it to lowercase, then use `split(" ")` and `join("-")` on it to hyphenate it.

Reviewer: ```suggestion
Create a `taskObj` object with an `id` property as the first property. For the value of the `id` property, retrieve the value of the `titleInput` field, convert it to lowercase, and then use the `split(" ")` and `join("-")` methods to hyphenate it.
```

Author: split(" ") and join("-") specifically means what to pass into those methods. I agree we should have them as `split()` and `join()`, since the hints show what to pass into both of them too.
